utils/permissions/src/main.rs

```diff
@@ -0,0 +1,108 @@
+use crate::doc_parser::enums::{EnumDoc, EnumDocParser};
```

Comment:
Let's add more "paragraph"-commentary here as well so that the code's broad strokes are more understandable.